examples/stable/elasticsearch/README.md

```diff
@@ -40,16 +40,16 @@ $ helm repo add elastic https://helm.elastic.co
 
 Then install the sample Elasticsearch application with the release name `my-release` in its own namespace
 `es-test` using the command below. Make sure you have the kanister controller running in namespace `kasten-io` which is the default setting in Elasticsearch charts. Otherwise, you will also have to set the `kanister.controller_namespace` parameter value to the respective kanister controller namespace in the following command:
-
+Please make a note that this command will be different if you are using helm version 3
```

Comment:
We can move this after the command and add an example for helm 3 command if available.

Reply:
Hi @pavannd1 ,
I have included the command now.